website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -13,7 +13,8 @@ a process via environment variables using
 [Consul Template markup][consul-templating-language].
 
 -> If you are running your applications in a Kubernetes cluster, we recommend
-   evaluating the [Vault Secrets Operator](/vault/docs/platform/k8s/vso) instead.
+  evaluating the [Vault Secrets Operator](/vault/docs/platform/k8s/vso) or
+  the [Vault Agent Sidecar Injector](/vault/docs/platform/k8s/injector) instead.
```

Comment:
```suggestion
  the [Vault Agent Sidecar Injector](/vault/docs/platform/k8s/injector) first.
```

OR

```suggestion
  the [Vault Agent Sidecar Injector](/vault/docs/platform/k8s/injector).
```

Reply:
Both are good to me! I think I prefer the version without. If we do make this change, it probably makes sense to change the "or" to an "and" too, e.g. instead of 'use X or Y', 'evaluate X and Y'

Reply:
Hey @SteVwonder ! I dug up the thread that prompted this to be changed. We're of the opinion that the other two approaches should be preferred if available, as they should be easier to use and more native to K8S.

There's no reason why the process orchestrator wouldn't work in K8S, and it's a supported usage of Agent, but we consider the other two options (Agent Injector and VSO) better, if they're available. I hope that makes sense!